qiime2/plugin/plugin.py

```diff
@@ -36,8 +36,9 @@ def yaml_representer(dumper, data):
         ]
         return dumper.represent_dict(items)
 
-    def __init__(self, name, version, website, package, citation_text=None,
-                 user_support_text=None):
+    def __init__(self, name, version, website, package,
+                 description=None, short_description=None,
```

Comment:
Can you put `short_description` and `description` as the last two keyword arguments in the signature? That'll be more backwards-compatible in case plugin developers are only using positional arguments (hopefully not, but you never know).

Also, can you swap the relative order of these new parameters so that `short_description` comes before `description`? That order makes more sense to me -- write the summary first and then the expanded description.